staging/src/k8s.io/component-base/OWNERS

```diff
@@ -0,0 +1,18 @@
+approvers:
+- luxas
+- sttts
+- mtaufen
+- DirectXMan12
+- liggitt
```

Comment:
please compare with KEP, I think the list differs.

Reply:
good catch, will do